failsafe/failsafe.py

```diff
@@ -87,6 +93,9 @@ def __init__(self, retry_policy=None, circuit_breaker=None):
                 self.circuit_breaker.record_failure()
 
                 if retry:
+                    if wait_for is not None:
```

Comment:
If we do it like I suggest above, I'd like to change this `if` to:

```
if wait_for > timedelta():
```

Basically wait_for would never be `None`, but it could be zero.

(but that's a detail)

Reply:
(maybe there is a nicer way of checking whether timedelta is > 0)